src/content/en/fundamentals/discovery-and-monetization/payment-request/index.md

```diff
@@ -754,8 +754,12 @@ do so, use `canMakePayment()`.
       request.show();
     }
 
-Note: This is only optional. Users can still add a new payment method in the
-Payment Request UI in general.
+Warning: In order to protect user privacy, calling <code>canMakePayment()</code>
+too often within a short time range causes [a "Quota exceeded"
```

Comment:
Something along the lines of: "At the time of writing the quota error is thrown if you query more than once every 30 minutes. This is Chrome specific and may change over time."